### PR TITLE
Add support for `modsec_rule_id` filter parameter

### PR DESCRIPTION
**Pull request by smaeda-ks**
https://developer.fastly.com/reference/api/waf/rules/#list-waf-rules

This is to support https://github.com/fastly/terraform-provider-fastly/issues/520

### CODE REVIEW

**Comment by smaeda-ks** [comment]
@Integralist

File: fastly/waf_rules.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ type ListWAFRulesInput struct {
 	FilterTagNames []string
 	// Limit the returned rules to a set by publishers.
 	FilterPublishers []string
+	// Limit the returned rules to a set by modsecurity rule IDs.
+	FilterModSecIDs []int
 	// Excludes individual rules by modsecurity rule IDs.
 	ExcludeMocSecIDs []int
```
Comment (Integralist) [inline fastly/waf_rules.go:55]: Out if interest this looks like a typo... right? Should be `ExcludeModSecIDs`? If so we should add a TODO to update and bump to a new major version.

**Review by Integralist — APPROVED** [review]
LGTM. Can we regenerate the fixtures to help validate the changes.

**Comment by smaeda-ks** [comment]
@Integralist Thanks. Added a test for this change.

**Pull request merged**